### ISSUE TITLE
The window loses control, IDK what's causing that. 

### ISSUE THREAD

**Issue by snehccurry** [Peticali/PythonBlurBehind#9]
I clicked on apply and then I tried to maximize to full screen back and forth three times, and then it became inactive, I'm using the example code for tkinter.

Also when I click on apply acrylic button, and then hit maximize back and forth, this is what I get:

![image](https://user-images.githubusercontent.com/59841219/179370465-fca62ed1-8f4a-4a22-8602-9df7ca0bafb6.png)

**Comment by Peticali** [comment]
yeah, windows 11 just made the acrylic bugs worse, unfortunately it's been a long time since I've worked with this code and it's totally discontinued, it needs to be rewritten, maybe you'll find something interesting in the glasstron project (electron with blur)

**Comment by sumeshir26** [comment]
Or, you could use [win32mica](https://pypi.org/project/win32mica/) which adds a Windows 11 mica blur to python.

**Comment by Peticali** [comment]
> Or, you could use [win32mica](https://pypi.org/project/win32mica/) which adds a Windows 11 mica blur to python.

for windows 11 would be ideal! I didn't know about this project, thanks
